internal/js_library/js_library.bzl

```diff
@@ -1,4 +1,4 @@
-# Copyright 2017 The Bazel Authors. All rights reserved.
+# Copyright 2020 The Bazel Authors. All rights reserved.
```

Comment:
note for future - my understanding is that copyright dates doesn't need to be updated on edit

Reply:
I've got the year back to the original one